nixos/modules/services/misc/gitlab.nix

```diff
@@ -339,9 +404,29 @@ in {
       };
 
       initialRootPassword = mkOption {
```

Comment:
I think `password` options don't provide much value, even for testing. Were a `passwordFile` option to document how easy it is to write something like `passwordFile = pkgs.writeText "insecure-password.txt" "just-for-testing";` I think that should be sufficient.

That being said, I have no idea how popular/unpopular my idea is :man_shrugging:

Reply:
Yeah, I agree. I went with this approach since it seemed like the norm for NixOS modules (like gitea) and this was hinted at in the old PR.

Reply:
You're entirely correct it is the norm for NixOS modules and this needs to be fixed! I'm hoping to remove a bunch of these options before `19.09`.

Reply:
Okay, then I'll just remove the old options.